<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSensingWidget</class>
 <widget class="QWidget" name="CSensingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CSensingWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QPushButton" name="Btn_Up">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../project/SimulatorCtl.qrc">
       <normaloff>:/img/up.png</normaloff>:/img/up.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="Btn_Left">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../project/SimulatorCtl.qrc">
       <normaloff>:/img/left.png</normaloff>:/img/left.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="Btn_Down">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../project/SimulatorCtl.qrc">
       <normaloff>:/img/down.png</normaloff>:/img/down.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="Btn_Right">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../project/SimulatorCtl.qrc">
       <normaloff>:/img/right.png</normaloff>:/img/right.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QToolButton" name="Btn_High">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>上升</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QToolButton" name="Btn_Low">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>下降</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../project/SimulatorCtl.qrc"/>
 </resources>
 <connections/>
</ui>
